schemas/2014-02-26/microsoft.visualstudio.json

```diff
@@ -21,11 +21,27 @@
           "type": "object",
           "properties": {
             "OperationType": {
-              "type": "string",
+              "anyOf": [
```

Comment:
We should use "oneOf", and maybe not for "string" type, as any "string" will match.